src/templates-idea/solarized-chandrian.theme.json

```diff
@@ -244,7 +244,7 @@
     },
     "Label": {
       "errorForeground": "{{red}}",
-      "infoForeground": "{{backgroundMild}}",
+      "infoForeground": "{{textMostMild}}BB",
```

Comment:
`backgroundMild` rendered very hard to read text, so changing to `textMostMild` with some transparency makes it much better